content/introduction/overview.md

```diff
@@ -30,57 +34,57 @@ Making it possible to download a file from many locations that aren’t managed
 - `C:\Users\Joe\My Documents\project_sprint_presentation.ppt`
 
 …identify a file by *where it’s located* — what computer it’s on and where on that computer’s hard drive it is. That doesn’t work if the file is in many places, though, like your neighbor’s computer and your friend’s across town.
 
-Instead of being location-based, IPFS addresses a file by *what’s in it*, or by its *content*. The content identifier above is a *hash* of the content at that address, which means it’s also unique to that content, even though it’s [relatively] short. It also allows you to verify you got what you asked for — bad actors can’t just hand you content that doesn’t match. (If hashes are new to you, check out [the concept guide on hashes]({{<relref "guides/concepts/hashes.md">}}) for a good introduction.)
+Instead of being location-based, IPFS addresses a file by *what’s in it*, or by its *content*. The content identifier above is a *cryptographic hash* of the content at that address. The hash is unique to the content that it came from, even though it may look short compared to the original content. It also allows you to verify that you got what you asked for — bad actors can’t just hand you content that doesn’t match. (If hashes are new to you, check out [the concept guide on hashes]({{<relref "guides/concepts/hashes.md">}}) for a good introduction.)
+
+The hash is unique to the content that it came from, even though it’s [relatively] short . It also allows you to verify that you got what you asked for — bad actors can’t just hand you content that doesn’t match. (If hashes are new to you, check out [the concept guide on hashes]({{<relref "guides/concepts/hashes.md">}}) for a good introduction.)
 
 <aside class="alert alert-info">
-  Why do we say “content” instead of “files” or “web pages” here? Because a content identifier can point to many different types of data. In order to make the storage & transmission of data more efficient, IPFS can break files up in to many smaller pieces — which then means the system needs some metadata to tie them back together. An IPFS address can refer to the metadata of just a single piece of a file, a whole file, a directory, a whole website, or even other kinds of data.
+  Why do we say “content” instead of “files” or “web pages” here? Because a content identifier can point to many different types of data, such as a single small file, a piece of a larger file, or metadata. (In case you don’t know, metadata is “data about the data.” You use metadata when you access the date, location, or file size of your digital pictures, for example.) So, an individual IPFS address can refer to the metadata of just a single piece of a file, a whole file, a directory, a whole website, or any other kind of content. For more on this, check out the [_How IPFS Works_](/introduction/how-ipfs-works/) part of these docs!
 </aside>
+Because the address of a file in IPFS is created from the content itself, links in IPFS can’t be changed. For example…
```

Comment:
address -> hash or CID. Address is not a good term to refer to hashes as it is not used anywhere in ipfs-land. If anything, "ipfs path".

Reply:
I would like to put a pin in this one. `address` is sprinkled through that doc, and to me, makes the overview easier to understand. I would like to come back to this in a separate PR for that _overview.md_ page.